### ISSUE TITLE
[A11y] Search Results - List - Checkboxes / Selecting Assets

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1970]
### Operating System

Windows

### Browser

Edge

### Affected page

Search

### Admin user?

Yes

### Issue Description

Checkboxes have no label. Very similar issues to #1930, but the checkboxes are always visible, so that's one less concern.

The big issue here is that something needs to be announced when Select All / None are activated.

### Suggested Fix

Add a "polite" aria-live region that tells users that something happened. At a minimum, "All assets selected" and "All assets deselected" should be stated to screen-reader users.  As mentioned in #1930, more information, and visual information, would probably benefit all users. Sighted users with low vision might never know what happened.

### Can anybody QA or does this need an expert?

Needs expert

### QA Steps

Probably needs a screen-reader user to test this out to make sure it's set up properly, has the right politeness level, etc.

**Comment by jechols** [comment]
This was not implemented correctly. The region appears to get updated properly, but it has `aria-hidden="true"` on it, which obviously prevents AT from knowing it exists in the first place:

`<p id="select-chosen" class="sr-only" aria-hidden="true">Selected All</p>`

**Comment by CGillen** [comment]
Fixed by #2155
